src/components/SocialBar/SocialBarIcon.st.css

```diff
@@ -0,0 +1,14 @@
+
+.root {
+  -st-states: theme(enum(light, dark));
+}
+
+.root:theme(light) .iconButton path {
+  fill: black;
+  stroke: none;
+}
+
+.root:theme(dark) .iconButton path {
+  fill: white;
+  stroke: none;
+}
```

Comment:
So,
After talking to the designer, we understood that we might need to change the styling of the component.
Dafna, our designer, will talk to Rob and Michal tomorrow to see what the correct direction to continue will be.

In any case, i see in the storybook that the `!important` was needed for this to work correctly.
However, with stylable there's no need to use `!important`.
Instead we write something like this:
```css
:import {
    -st-from: "../IconButton/IconButton.st.css";
    -st-default: IconButton;
}

.root {
  -st-states: theme(enum(light, dark));
}

.root .iconButton {
  -st-extends: IconButton;
}

.root:theme(light) .iconButton {
  -st-mixin: IconButton(
    IconColor #fff
  )
}

.root:theme(dark) .iconButton {
  -st-mixin: IconButton(
    IconColor #000
  )
}
```

Reply:
added extending IconButton